src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/VirtualSecureModeEnclaveProviderBase.cs

```diff
@@ -256,7 +256,9 @@ private bool VerifyHealthReportAgainstRootCertificate(X509Certificate2Collection
             // A certificate is simply used as a key pair consisting of a public and private key. This is by design.
 
             // CodeQL [SM00395] By design. Always Encrypted certificates should not be checked.
+            #pragma warning disable IA5352
```

Comment:
I think the CodeQL comment need to be right above the line it applies to. I'd swap 258 and 259.

```suggestion
            #pragma warning disable IA5352
            // CodeQL [SM00395] By design. Always Encrypted certificates should not be checked.
```